moja-kuhinjica-frontend/src/components/button/DayButton.tsx

```diff
@@ -0,0 +1,28 @@
+import React from 'react'
+import styles from './DayButton.module.scss'
+
+interface IDayButtonProps {
+    active: number
+    setActive: (param: number) => void
+    buttonNumber: number
+    content: string
+}
+export const DayButton = ({
+    active,
+    setActive,
+    buttonNumber,
+    content,
+}: IDayButtonProps) => {
+    return (
+        <button
+            onClick={() => setActive(buttonNumber)}
+            className={
+                active === buttonNumber
```

Comment:
This comparison can be done in parent component, so you don't really need this comparison here

Reply:
Done